samples/sample-bugsnag/gradle.properties

```diff
@@ -13,6 +13,4 @@ android.useAndroidX=true
 
 org.gradle.jvmargs=-Xmx4g
 
-kotlin.native.cacheKind.iosX64=none
-kotlin.native.cacheKind.iosSimulatorArm64=none
-
+kotlin.native.cacheKind=none
```

Comment:
👋 Hello folks, is this the new recommended way of disabling K/N caches for CrashKiOS? Should the [website]() be updated as well?

Thank you!

[website]: https://crashkios.touchlab.co/docs/crashlytics/#step-2---add-crashkios